### DISCUSSION TITLE
Can two field names be assigned on the igx column?

### DISCUSSION THREAD

**Discussion by chaklasiyanikunj** [IgniteUI/igniteui-angular#13768] [Q&A] [answered]
I have multiple fields on my grid. Here is the [code](https://stackblitz.com/edit/t1tdrq?file=src%2Fapp%2Fgrid%2Fgrid-search-sample%2Fgrid-search-sample.component.html)

``` \
<igx-column [field]="'IndustrySector'" dataType="string" [sortable]="true">
      <!-- I need to search with this two below fields-->
      <ng-template igxCell let-cell="cell">
        <div class="row">
          <div>
            <b> {{ cell.row.data.IndustrySector }} </b>
          </div>
          <div>
            {{ cell.row.data.IndustryGroup }}
          </div>
        </div>
      </ng-template>
    </igx-column>
    <!-- <igx-column [field]="'IndustryGroup'" dataType="string" [sortable]="true"></igx-column> -->
    <igx-column
      [field]="'SectorType'"
      dataType="string"
      [sortable]="true"
    ></igx-column>
```
I need to perform the search operations. It only works with whichever name I mention in [field]="'IndustrySector.'" It's not able to search with a second value like IndustryGroup.

Is there any solution for multiple names being in one igx column? Then, What should be the best approach for searching?

**Comment by ddaribo** [comment, marked as answer]
Hi @chaklasiyanikunj,

According to the Grid Searching topic [section about known limitations](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/search#known-limitations), I am afraid that searching in cells with a template is listed as such.

By the way, this question and the code-snippet's data properties are identical to a query from [this recent forum thread](https://www.infragistics.com/community/forums/f/ignite-ui-for-angular/125925/ignite-ui-angular-angular-components-grid-search). 
Please, check out my reply in the forum, where there is also a workaround suggested. Of course, depending on the cell template layout and styles, not everything could be achieved via CSS styles, nevertheless, this would be the approach to adopt in order to have the Searching feature work over the `IndustrySector` column's cells.
